### ISSUE TITLE
Why use the binary_cross_entropy as the reconstruction loss of variational_autoencoder?

### ISSUE THREAD

**Issue by alance123** [prabhuomkar/pytorch-cpp#34]

**Comment by prabhuomkar** [comment]
Binary Cross Entropy (BCE) as reconstruction loss function is in place to calculate how many “bits” of information are preserved in the reconstruction compared to the original.

For more reference about loss functions and auto encoders, read here: [https://yaledatascience.github.io/2016/10/29/autoencoders.html](https://yaledatascience.github.io/2016/10/29/autoencoders.html)

**Comment by alance123** [comment]
Thank you.
